modules/aggregation-layers/src/gpu-grid-layer/gpu-grid-layer.js

```diff
@@ -233,6 +240,9 @@ export default class GPUGridLayer extends CompositeLayer {
   }
 
   renderLayers() {
+    if (!GPUGridAggregator.isSupported(this.context.gl)) {
```

Comment:
I wonder is there other solution, which only need check once instead of checking in multiple places?

Reply:
Addressed, added a state variable. I was actually looking for a way, so layer-manager doesn't call updates on this, but I can only do that for sublayers by returning `null`, couldn't find a way to do it for current layer.